### PR TITLE
updated to the latest release of d language

### PR DESCRIPTION
**Pull request by abhi18av**

### CODE REVIEW

**Comment by abhi18av** [comment]
@conda-forge-admin could you please take a look at this one ?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "dmd" %}
-{% set version = "2.065.0" %}
+{% set version = "2.079.0" %}
 {% set sha256 = "f3f4c8f6c2e5cdd0ac335e56613663bee075a1e797ffbb55035d6c767d7bc585" %}
```
Comment (jakirkham) [inline recipe/meta.yaml:3]: Checksum probably needs to be updated.

**Comment by jakirkham** [comment]
Thanks @abhi18av. LGTM with a small change above.

In the future would ping `@conda-forge/core`. `@conda-forge-admin` is a bot, which has yet to learn how to review PRs (though that's not off the table ;).

**Comment by jakirkham** [comment]
@conda-forge-admin, please re-render.

**Pull request closed without merging**